sites/partners/pages/settings/index.tsx

```diff
@@ -200,6 +230,39 @@ const Settings = () => {
         saveQuestion={saveQuestion}
         isLoading={isCreateLoading || isUpdateLoading}
       />
+      <Modal
+        open={!!cloneDrawerOpen}
+        title={t("t.copy")}
+        ariaDescription={t("listings.listingIsAlreadyLive")}
+        onClose={() => setCloneDrawerOpen(null)}
+        actions={[
+          <Button
+            type="button"
+            styleType={AppearanceStyleType.primary}
+            onClick={() => {
+              saveQuestion({ ...cloneDrawerOpen, text: `Copy of ${cloneDrawerOpen.text}` }, "add")
```

Comment:
With the addition of `Copy of` the new records are now getting placed at the top of the list (at least locally because there are no preferences starting with A, B, or C). Is that ok? I know the next step is to highlight the row, but we didn't really discuss if the placement of the cloned preference should move

Reply:
@slowbot Do you have a preference / whether it should happen now or later?